lib/mini_autobot/google_sheets.rb

```diff
@@ -0,0 +1,89 @@
+module MiniAutobot
+
+  # Class that supports integration with Google Sheets
+  # See documentation here: https://developers.google.com/drive/v3/web/about-auth to set up OAuth 2.0
+  # access to your sheets. Place the info in google_drive_config.json in your config/mini_autobot folder
+  #
+  # To use, use the -g or --google_sheets parameter with the id of your sheet
+  # Example: -g 1gWbpPq7rrTdNtSJSN2ljDlPE-okf0w3w7ai5PlhY8bk
+  class GoogleSheets
+    require 'rubygems'
+    require 'google/api_client'
+    require 'google_drive'
+    require 'json'
+
+    def initialize(args)
+      @args = args
+      @session = session
+      @spreadsheet = spreadsheet
+      @worksheet = worksheet
+      @automation_serial_column = automation_serial_column
+      @selected_browser_column = selected_browser_column
+    end
+
+    # Updates all cells with the value provided that have the corresponding key in the Automation Serial Column
+    # At the end of your test, place the following line:
+    # Example:
+    # MiniAutobot.google_sheets.update_cells('Auto Pass', 'HP-1') if MiniAutobot.settings.google_sheets?
+    def update_cells(value, key)
+      rows = target_rows(key)
+      rows.each do |row|
+        @worksheet[row, @selected_browser_column] = value
+      end
+      @worksheet.save
+    end
+
+    private
+
+    def session
+      GoogleDrive.saved_session(@args[:session])
+      #GoogleDrive.saved_session(MiniAutobot.root.join('config/mini_autobot', 'google_drive_config.json'))
```

Comment:
Minor, but can you remove these lines instead of commenting out? You can remove them in a new commit, if you wish to retain the history.